tasks/main.yml

```diff
@@ -14,10 +14,36 @@
     __mssql_sqlcmd_login_cmd: null
 
 - name: Link the deprecated accept_microsoft_sql_server_2019_standard_eula fact
-  set_fact:
-    mssql_accept_microsoft_sql_server_standard_eula: >-
-      {{ mssql_accept_microsoft_sql_server_2019_standard_eula }}
   when: mssql_accept_microsoft_sql_server_2019_standard_eula is defined
+  block:
+    - name: Print that the variable is deprecated
+      debug:
+        msg: >-
+          "The accept_microsoft_sql_server_2019_standard_eula variable is
+          deprecated and will be removed in a future version. Edit your playbook
+          to use the mssql_accept_microsoft_sql_server_standard_eula variable
+          instead."
```

Comment:
If you are using the yaml string `>` or `|` (with and without the `-` modifier) you do not have to quote the string.

Reply:
Good to know, I've opened #86 for this and will fix all occurrences in one go as I have many of them throughout the role.